tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs/test_stub_docstring_creation[numpydoc-NUMPYDOC].sdsstub

```diff
@@ -193,6 +193,18 @@ fun numpyNamedResultWithoutType() -> namedResult: String
 @PythonName("numpy_named_result_without_type_inferred")
 fun numpyNamedResultWithoutTypeInferred() -> namedResult: String
 
+/**
+ * @example
+ * pipeline example {
+ *     // print(1+1)
+ *     // x = 2 - 1
+ *     // print(x)
+ * }
```

Comment:
The issue description was a little unclear here, unfortunately. The "Examples" section contains a list of examples, and there should be one `@example` block for each of them:

```
 * @example
 * pipeline example {
 *     // print(1+1)
 * }
 *
 * @example
 * pipeline example {
 *     // x = 2 - 1
 *     // print(x)
 * }
```

Later examples may depend on earlier ones, but you can ignore that.

Reply:
Adjusted the code, now we create multiple example blocks.